client/app/app.component.ts

```diff
@@ -39,4 +57,21 @@ export class AppComponent {
 	setActiveTab(tabName){
 		this.activeTab = tabName;
 	}
+
+	changeNav(){
```

Comment:
didn't see it last time.
I think the appropriate name for this function now is 'LoadNavbar'.
(It was 'changed' on our original plan, but now it only loads once when the page first loads)

also don't be afraid to use longer variable names - like `LoadNavbar` instead of `LoadNav` to make the code more readable. ;)